### ISSUE TITLE
more information to compare runs?

### ISSUE THREAD

**Issue by aminnj** [jkguiang/AutoDQM#5]
Hi,

Thanks for making a nice piece of software to simplify the life of a lowly DQM shifter. 

It might be nice to provide some more information about data and reference runs to help the shifter get a better idea about some differences. For example, some differences are expected when instantaneous luminosities are different.

Another useful feature would be to recommend a reference run automatically. Some simple requirement could be that the reference run is the latest run before the data run which has been marked "GOOD" in the run-registry (bonus: and has a similar instantaneous luminosity).

I'll dump some information here and if you find it useful/easy to add, that would be great. The [run-registry twiki](https://twiki.cern.ch/twiki/bin/view/CMS/DqmRrApi) has information about the run-registry API which can also access WBM information about individual runs. On lxplus/in the CERN network, this is as easy as the example on the twiki. For example
```bash
cp /afs/cern.ch/user/v/valdo/public/rhapi.py .
```
and
```python
from rhapi import DEFAULT_URL, RhApi

api = RhApi(DEFAULT_URL, debug = False)

cols = ["runnumber",
        "events",
        "triggers",
        "starttime",
        "initlumi",
        "delivered",
        "recorded",
        "triggerbase",
        "endlumi"]
q = "select {} from wbm.runs r where r.runnumber>:minrun".format(",".join("r."+x for x in cols))
p = {"minrun": 319950}
qid = api.qid(q)

data = api.json(q,p)["data"]
from prettytable import PrettyTable
p = PrettyTable(field_names=cols)
for r in data:
    p.add_row(r)
print(p.get_string())
```
prints some information about some of the latest runs (including luminosities). The table for the CSC run registry where you can see the "GOOD" (or "NOTSET") flag is `runreg_csc.datasets`. As an example of what I would personally consider useful, I have a bookmarklet to make a summary table with links to offline DQM and autoDQM with automatic reference runs (like above, but I don't check luminosity) [here](https://gist.github.com/aminnj/63891461e43fd42615cabc4e82429544).

Unfortunately, as far as I can tell, the run-registry doesn't have an easy API to mark things as good, otherwise I'd want to move in the direction of having a single page that DQM shifters can pull up for their entire job.

Thanks again for this tool.

Nick

**Comment by jkguiang** [comment]
#7

**Comment by jkguiang** [comment]
Implemented here: #7
